src/common/constants.js

```diff
@@ -6,7 +6,7 @@ export const LOG_SOCKET_IO = false;
 export const KEEP_DECKS_UNSHUFFLED = false;
 export const DISABLE_TURN_TIMER = false;
 export const DISABLE_AI = false;
-export const DISPLAY_HEX_IDS = false;
+export const DISPLAY_HEX_IDS = true;
```

Comment:
Is this the constant to display the hex IDs on the board? If so why is it true now?

Reply:
oops